CONTRIBUTING.md

```diff
@@ -14,6 +14,7 @@ If you want to submit your own contributions, follow these steps:
 
 * Fork the Cumulus Dashboard repo
 * Create a new branch from the branch you'd like to contribute to
+* Establish that everything is working before you write code. See ['Getting Started for beginners'](#getting-started-for-beginners) below.
```

Comment:
Optional Change: I suggest 'Verify' or 'Ensure' instead of 'Establish' since nothing is being created.